examples/js/loaders/GLTFLoader.js

```diff
@@ -531,6 +531,138 @@ THREE.GLTFLoader = ( function () {
 	 *
 	 * Specification: https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_materials_pbrSpecularGlossiness
 	 */
+
+	/**
+	 * A sub class of THREE.StandardMaterial with some of the functionality 
+	 * changed via the `onBeforeCompile` callback
+	 * @pailhead
+	 */
+
+	function SpecularGlossinessPbrMaterial( params ) {
```

Comment:
I think we discussed possible names if a spec/gloss PBR material were added to three.js once before, and MeshStandardSGMaterial was the candidate? Perhaps let's use that.